ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentIdProvider.scala

```diff
@@ -13,15 +13,15 @@ trait ComponentIdProvider {
 }
 
 class DefaultComponentIdProvider(configs: Map[ProcessingType, ComponentsUiConfig]) extends ComponentIdProvider {
-  override def createComponentId(processingType: ProcessingType, name: String, componentType: ComponentType): ComponentId =
-    //We assume that base component's id can't be overridden
-    if (ComponentType.isBaseComponent(componentType)) {
-      ComponentId.forBaseComponent(componentType)
-    } else {
-      val defaultComponentId = ComponentId.default(processingType, name, componentType)
-      val overriddenComponentId = getOverriddenComponentId(processingType, name, defaultComponentId)
-      overriddenComponentId
-    }
+  override def createComponentId(processingType: ProcessingType, name: String, componentType: ComponentType): ComponentId = {
+    val defaultComponentId = ComponentId.default(processingType, name, componentType)
+
+    //We assume that base nad fragment component's id can't be overridden
```

Comment:
I think we should add 'currently' for fragments. It's just an implementation limitation, not sth we're doing by design? What's more - maybe we should throw some exception/warning if use tries to do it?

Reply:
I'v added throwing exception.